command/ca/ca.go

```diff
@@ -123,6 +128,19 @@ unit suffix, such as "300ms", "-1.5h" or "2h45m". Valid time units are "ns",
 "us" (or "µs"), "ms", "s", "m", "h".`,
 	}
 
+	offlineFlag = cli.BoolFlag{
+		Name: "offline",
+		Usage: `Creates a certificate without contacting the certificate authority. Offline mode
+requires the CA configuration file given using the '--ca-config>' flag.`,
```

Comment:
I would be more clear here. It doesn't actually require that flag, because the flag can be inferred.